pulumi/grapl/Pulumi.local-grapl.yaml

```diff
@@ -29,6 +29,7 @@ config:
   grapl:cloudsmith-repository-name: grapl/testing
   grapl:artifacts:
     "firecracker_kernel.tar.gz": "firecracker-v1.0.0-kernel-4.14.174"
+    "firecracker_rootfs.tar.gz": "20220329205538-5ba93c6"
```

Comment:
Do we have a way to keep this up-to-date, or is this just manual at this point?

Reply:
Yeah, I was just asking for general knowledge purposes; I'm not suggesting any additional work for this PR.